File: problems/hacker_rank/month/Day8.java
package hacker_rank.month;

import java.util.HashMap;
import java.util.Map;
import java.util.Scanner;

class Day8 {

    public static void main(String[] arg){

        Map<String,Integer> map = new HashMap<>();

        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        for(int i = 0; i < n; i++){
            String name = in.next();
            int phone = in.nextInt();
            // Write code here
            map.put(name,phone );
        }
        while(in.hasNext()){
            String s = in.next();
            if (map.containsKey(s)){
                System.out.println(s+"="+map.get(s));
            } else {
                System.out.println("Not found");
            }
        }
        in.close();

    }

}

File: problems/hacker_rank/month/Day8.kt
package hacker_rank.month


fun main(args: Array<String>) {

    val time = readLine()?.trim()?.toInt() ?: 0

    val map:HashMap<String,Int> = hashMapOf()

    try{
        for (n in 0 until time) {
            val add = readLine() ?: ""
            val li = add.split(" ")
            map[li[0]] = li[1].toInt()
        }

        for (n in 0 until time) {
            val key = readLine() ?: ""
            if (map.containsKey(key)){
                println("$key=${map[key]}")
            } else{
                println("Not found")
            }
        }
    } catch (e:Exception){

    }

}

